### PR TITLE
codestyle fixes for swiftlint

### PR DESCRIPTION
**Pull request by spielplatz**
### Work
- Fixes: just codestyle
- Description: I want to get rid of fixing swiftlint errors after each update

### Checklist

- [ ] Updated CHANGELOG.md

### CODE REVIEW

File: BitmovinAnalyticsCollector/Classes/Collector/data/SimpleEventDataDispatcher.swift [hunk 1]
```diff
@@ -20,9 +20,9 @@ class SimpleEventDataDispatcher: EventDataDispatcher {
                 guard let events = self?.events.enumerated().reversed() else {
                     return
                 }
-                for (i, eventData) in events {
+                for (imdex, eventData) in events {
```
Comment (Tigraine) [inline BitmovinAnalyticsCollector/Classes/Collector/data/SimpleEventDataDispatcher.swift]: shouldn't this be `index`?

File: BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift [hunk 2]
```diff
@@ -79,20 +78,21 @@ class AVPlayerAdapter: NSObject, PlayerAdapter {
...
-    
+
     private func errorOccured(error: NSError?) {
         let errorCode = error?.code ?? 1
         let errorMessage = error?.localizedDescription ?? "Unkown"
-        
-        stateMachine.transitionState(destinationState: .error, time: player.currentTime(), data: [BitmovinAnalyticsInternal.ErrorCodeKey: errorCode, BitmovinAnalyticsInternal.ErrorMessageKey: errorMessage])
+
+        stateMachine.transitionState(destinationState: .error,
+                                     time: player.currentTime(), data: [BitmovinAnalyticsInternal.ErrorCodeKey: errorCode,
```
Comment (soberman) [inline BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift]: `time:` parameter from the new line but not he `data:`?

File: BitmovinAnalyticsCollector/Classes/Collector/data/SimpleEventDataDispatcher.swift (hunk 1 above)
Comment (spielplatz) [reply BitmovinAnalyticsCollector/Classes/Collector/data/SimpleEventDataDispatcher.swift]: yes, it's a typo, should be index. Swiftlint demands variable names not shorter than 3 chars. So, you can name it anything longer than in ;)

File: BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift (hunk 2 above)
Comment (spielplatz) [reply BitmovinAnalyticsCollector/Classes/AVPlayer/Classes/AVPlayerAdapter.swift]: will be mich better readable, yes. I just shortened the line length.

**Pull request merged**